.gitignore

```diff
@@ -15,3 +15,4 @@ constants.local.php
 *.autosave
 *.sass-cache
 sass.sh
+Docker/key.gpg
```

Comment:
What is this key for?

Reply:
Apparently it was my key and I have no idea how it ended up their. I though it was a key from the build process or something related to FreshRSS. Gonna drop that commit.